src/app/home/modals/add-category-modal/add-category-modal.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, inject } from '@angular/core';
+import { FormGroup, NonNullableFormBuilder, Validators } from '@angular/forms';
+import { CategoryFormModel } from '@home/models/form.model';
+import { MatDialogRef } from '@angular/material/dialog';
+
+@Component({
+  selector: 'app-add-category-modal',
+  templateUrl: './add-category-modal.component.html',
+  styleUrl: './add-category-modal.component.scss'
+})
+export class AddCategoryModalComponent {
+  categoryForm: FormGroup<CategoryFormModel>;
+  userId: string = JSON.parse(localStorage.getItem('user')!).id;
```

Comment:
it's private

Reply:
and I don't see how you used it 
you can add it to form and then I'll have object with this data 
this.categoryForm = this.fb.group({
      name: ['', Validators.required],
      capacity: [0, [Validators.required, Validators.pattern(/^\d+$/)]],
userId: [this.userId]
    });